config/prod/shinyserver-pro-prod-alb-v2.yaml

```diff
@@ -0,0 +1,22 @@
+# Provision a load balancer for shiny.synapse.org
+template_path: remote/managed-alb-https-v2.yaml
+stack_name: shinyserver-pro-prod-alb-v2
```

Comment:
you'll also need to add a `dependencies:` section and list all the stacks you reference parameters from.